sdk/core/perf/README.md

```diff
@@ -50,7 +50,7 @@ The next options can be used for any test:
 | ---------- | ---        | ---| ---| --- |
 | Duration   | -d, --duration   | Duration of the test in seconds                  | 10    | -d 5
 | Host       | --host           | Host to redirect HTTP requests                   | NA    | --host=https://something.com
-| Insecure   | --insecure       | Allow untrusted SSL certs                        | false | --insecure=true
+| Insecure   | --insecure       | Allow untrusted SSL certs                        | false | --insecure
```

Comment:
Why is this one different from the other bool ones like statistics or nocleanup?

Reply:
the other two are not enabled yet.  I would probably remove them from the docs or add a note about is not supported yet